watchtower/standalone.go

```diff
@@ -169,12 +170,21 @@ func (w *Standalone) createNewHiddenService() error {
 	onionCfg := tor.AddOnionConfig{
 		VirtualPort: DefaultPeerPort,
 		TargetPorts: listenPorts,
-		Store:       tor.NewOnionFile(w.cfg.WatchtowerKeyPath, 0600),
-		Type:        w.cfg.Type,
+		Store: tor.NewOnionFile(
+			w.cfg.WatchtowerKeyPath, 0600,
+			w.cfg.EncryptKey, w.cfg.KeyRing,
+		),
+		Type: w.cfg.Type,
 	}
 
 	addr, err := w.cfg.TorController.AddOnion(onionCfg)
 	if err != nil {
+		if err == tor.ErrEncryptedPrivateKey {
+			return fmt.Errorf("It appears the Tor private key is " +
```

Comment:
Nit: could extract the error into a var and reuse it.

Reply:
@wpaulino I'm not sure I follow what you mean. Could you please elaborate?

Reply:
Make a concrete error containing that message similar to `ErrEncryptedPrivateKey` so that we don't have the same string message twice throughout the codebase.

Reply:
@wpaulino Sure that makes sense. Do you have a recommendation on where to put it? Forgive me if I'm missing something obvious but I'm struggling to find a good spot to define the error where both the `lnd` package and `watchtower` package can reference it. 😅

Reply:
`lncfg/tor.go` seems like the best place for it.

Reply:
@wpaulino I can't import the `lncfg` package into the `watchtower` package because that will cause circular imports.  

`lncfg` already imports `watchtower`:
https://github.com/lightningnetwork/lnd/blob/master/lncfg/watchtower.go#L3

Reply:
Ah, so we want to return the error from the caller of `watchtower.Standalone.Start` instead. Having the error in `lncfg` would work then.

Reply:
💯 yes, of course. Just pushed that change. PTAL. Thanks for holding my hand through that one 😉